administration/src/models/CardCreationModel.ts

```diff
@@ -0,0 +1,8 @@
+import {CardType} from "./CardType";
+
+export interface CardCreationModel {
```

Comment:
This model is only used in EakGeneation.ts. That means you can simply inline the interface there. I think right now there is no need for a separate `models` directory.